env0/data_project_test.go

```diff
@@ -0,0 +1,57 @@
+package env0
+
+import (
+	"testing"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestUnitProjectData(t *testing.T) {
+	resourceType := "env0_project"
+	resourceName := "test_project"
+	resourceFullName := dataSourceAccessor(resourceType, resourceName)
+	project := client.Project{
+		Id:          "id0",
+		Name:        "my-project-1",
+		CreatedBy:   "env0",
+		Role:        "role0",
+		Description: "A project's description",
+	}
+
+	projectByName := map[string]string{
+		"name": project.Name,
+	}
+
+	projectById := map[string]string{
+		"id": project.Id,
+	}
+
+	runScenario := func(input map[string]string, mockFunc func(mockFunc *client.MockApiClientInterface)) {
+		testCase := resource.TestCase{
+			ProviderFactories: testUnitProviders,
```

Comment:
remove this line, we now override this

Reply:
https://github.com/env0/terraform-provider-env0/pull/98/commits/02e4bf744a7880bffbaa339117dd52a67b313c88